src/BlazorUI/Bit.BlazorUI/Styles/Theme/colors.scss

```diff
@@ -10,6 +10,9 @@
     --clr-secondary-darker: var(--clr-ntr-gray30);
     //background color
     --clr-background: var(--clr-ntr-white);
+    //general
+    --clr-gnr-red: #A4262C;
```

Comment:
I think to better use the `error` name instead of `general red`. I use this name in[ this PR](https://github.com/bitfoundation/bitplatform/pull/3875)